Lib/test/test_ssl.py

```diff
@@ -2215,7 +2218,7 @@ def wrap_conn(self):
                     if support.verbose and self.server.chatty:
                         sys.stdout.write(" client cert is " + pprint.pformat(cert) + "\n")
                     cert_binary = self.sslconn.getpeercert(True)
-                    if support.verbose and self.server.chatty:
+                    if support.verbose and self.server.chatty and cert_binary != None:
```

Comment:
@tiran This would have been needed due to the recent CVE fix, right? Why didn't we have to put it in already? Or did we not add tests for this case.

Reply:
Which CVE are you referring to? The CRL DP NULL pointer CVE? That shouldn't be related here.

With TLS 1.3 ``getpeercert`` returns None when the client has not send a client auth cert yet. This should only occur when the server-side has enable post handshake auth.

Reply:
I turned chatty on temporarily while testing and hit this code path which isn't normally hit.  I don't think it's new behavior

Reply:
this change has been removed